cloud-sql/sql-server/sqlalchemy/Dockerfile

```diff
@@ -16,6 +16,18 @@
 # https://hub.docker.com/_/python
 FROM python:3.10-buster
 
+RUN apt-get update
+RUN apt install unixodbc-dev -y
+
+# Add SQL Server ODBC Driver 17 for Ubuntu 18.04
+RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+RUN curl https://packages.microsoft.com/config/ubuntu/18.04/prod.list > /etc/apt/sources.list.d/mssql-release.list
+RUN apt-get update
```

Comment:
@jsimonweb do we need this second `RUN apt-get update`?

Reply:
Yes. Without this command the build of the sample app container fails with the error:
E: Unable to locate package msodbcsql17
The command '/bin/sh -c ACCEPT_EULA=Y apt-get install -y --allow-unauthenticated msodbcsql17' returned a non-zero code: 100

Reply:
Perfect! Figured there was a reason but just wanted to double-check. LGTM